aodndata/moorings/burst_average.py

```diff
@@ -0,0 +1,447 @@
+        if var_units:
+            output_var_mean['units'] = var_units
+            output_var_min['units'] = var_units
+            output_var_max['units'] = var_units
+            output_var_sd['units'] = var_units
+
+        var_stdname = getattr(input_var_object, 'standard_name', '')
+        if var_stdname != '':
+            output_var_num_obs['standard_name'] = "%s number_of_observations" % var_stdname
+
+        # set up var values
+        output_var_mean['_data'] = np.ma.masked_invalid(burst_vars[var]['var_mean'])
+        output_var_min['_data'] = np.ma.masked_invalid(burst_vars[var]['var_min'])
+        output_var_max['_data'] = np.ma.masked_invalid(burst_vars[var]['var_max'])
+        output_var_sd['_data'] = np.ma.masked_invalid(burst_vars[var]['var_sd'])
+        output_var_num_obs['_data'] = np.ma.masked_invalid(burst_vars[var]['var_num_obs'])
+
+    # set up original varatts for the following dim, var
+    varnames = dimensionless_var
+    varnames.append('TIME')
+    for var in varnames:
+        template.variables[var].update(input_netcdf_obj[var].__dict__)
+
+    time_comment = '%s. Time stamp corresponds to the middle of the burst measurement.' % getattr(
+        input_netcdf_obj['TIME'], 'comment', '')
+    template.variables['TIME']['comment'] = time_comment.lstrip('. ')
+
+    # append original gatt to burst average gatt
+    if hasattr(input_netcdf_obj, 'comment'):
+        template.global_attributes['comment'] = input_netcdf_obj.comment
+
+    gatt = 'history'
+    template.global_attributes[gatt] = '{old}. Created {new}'.format(old=getattr(input_netcdf_obj, gatt, ''),
+                                                                     new=time.ctime(time.time())
+                                                                     ).lstrip('. ')
+
+    gatt = 'abstract'
+    template.global_attributes[gatt] = '{old}. {new}'.format(old=getattr(input_netcdf_obj, gatt, ''),
+                                                             new='Data from the bursts have been cleaned and averaged '
+                                                                 'to create this product.'
+                                                             ).lstrip('. ')
+
+    # add burst keywords
+    gatt = 'keywords'
+    keywords_burst = 'AVERAGED, BINNED'
+    template.global_attributes[gatt] = '{old}, {new}'.format(old=getattr(input_netcdf_obj, gatt, ''),
+                                                             new=keywords_burst
+                                                             ).lstrip(', ')
+
+    # add values to variables
+    template.variables['TIME']['_data'] = np.ma.masked_invalid(time_burst_vals)
+
+    github_comment = (' Product created with https://github.com/aodn/python-aodndata'
+                      '/blob/{v}/aodndata/moorings/burst_average.py'.format(v=__version__)
+                      )
+    template.global_attributes['lineage'] += github_comment
```

Comment:
Because I don't think it will work when the code is deployed in RC or prod, as that is done from a pre-built wheel rather than a local clone of the git repo.

Reply:
makes sense. Maybe we could get @lwgordonimos 's opinion on the best way to do this? maybe a core functionality ?

Reply:
Maybe one day we'll have these packages in PyPI, and then referring to the package name and version will be sufficient?

In any case, I think this is good enough for now.